xsd/netex_framework/netex_reusableComponents/netex_deckPlan_support.xsd

```diff
@@ -0,0 +1,661 @@
+            services, Bus stops and stations,
+            Rail transport, Railway stations and track, Train services, Underground trains,
+            Business and industry, Transport, Air transport , Ports and maritime
+            transport, Public transport,
+            Rail transport, Roads and Road transport
+          </Category>
+					<Project>CEN TC278 WG3 SG9.</Project>
+				</Subject>
+				<Title>NeTEx DECK PLAN types.</Title>
+				<Type>Standard</Type>
+			</Metadata>
+		</xsd:appinfo>
+		<xsd:documentation>DECK PLAN identifier  types</xsd:documentation>
+	</xsd:annotation>
+	<!-- ==== DECK PLAN==================================================== -->
+	<xsd:complexType name="deckPlanRefs_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a list of DECK PLANs.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="oneToManyRelationshipStructure">
+				<xsd:sequence>
+					<xsd:element ref="DeckPlanRef" maxOccurs="unbounded"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:simpleType name="DeckPlanIdType">
+		<xsd:annotation>
+			<xsd:documentation>Type for identifier of a DECK PLAN.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="ObjectIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="DeckPlanRef" type="DeckPlanRefStructure" abstract="false" substitutionGroup="VersionOfObjectRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a DECK PLAN. +v2.0</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="DeckPlanRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a reference to a DECK PLAN.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:simpleContent>
+			<xsd:restriction base="VersionOfObjectRefStructure">
+				<xsd:attribute name="ref" type="DeckPlanIdType" use="required">
+					<xsd:annotation>
+						<xsd:documentation>Identifier of a DECK PLAN.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<!-- ==== DECK  ==================================================== -->
+	<xsd:complexType name="deckRefs_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a list of DECKs.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="oneToManyRelationshipStructure">
+				<xsd:sequence>
+					<xsd:element ref="DeckRef" maxOccurs="unbounded"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:simpleType name="DeckIdType">
+		<xsd:annotation>
+			<xsd:documentation>Type for identifier of a DECK.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="ObjectIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="DeckRef" type="DeckRefStructure" substitutionGroup="VersionOfObjectRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a DECK. +v2.0</xsd:documentation>
```

Comment:
@skinkie  General question: for a new file: could/should the "+v2.0" better be put at the level of the file, somehow, or should it be added "everywhere"? And what is a reasonable "everywhere"?

Reply:
general discussion.